src/io/flutter/view/FlutterView.java

```diff
@@ -237,6 +238,16 @@ protected void perform(AnActionEvent event) {
   }
 }
 
+class ToggleInspectModeAction extends AbstractToggleableAction {
+  ToggleInspectModeAction(@NotNull FlutterView view) {
+    super(view, "Toggle Inspect Mode", "Toggle Inspect Mode", AllIcons.Actions.Menu_find);
+  }
+
+  protected void perform(AnActionEvent event) {
+    view.getFlutterApp().callBooleanExtension("ext.flutter.widgetInspector", isSelected(event));
```

Comment:
from the flutter tool, press 'i' to enter inspect mode, press 'i' again to exit inspect mode.